src/v/coproc/script_context.cc

```diff
@@ -201,7 +201,7 @@ script_context::get_reader(const ss::lw_shared_ptr<ntp_context>& ntp_ctx) {
     const storage::offset_stats os = ntp_ctx->log.offsets();
     return storage::log_reader_config(
       next_read,
-      os.committed_offset,
+      os.dirty_offset,
```

Comment:
any reason this isn't merged into the previous commit?

Reply:
The rational here was that changing this fundamentally changes how copro behaves, and this change isn't really connected to the off-by-one bugfix at all.